File: android/app/modules/mobile.android/src/main/java/com.multiapk.modules.mobile.android/AndroidFragment.java
package com.multiapk.modules.mobile.android;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.Toast;

import com.mlibrary.multiapk.core.hotpatch.Hotpatch;
import com.multiapk.library.base.MFragment;

import java.io.File;

public class AndroidFragment extends MFragment {
    @SuppressLint("InflateParams")
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View contentView = inflater.inflate(R.layout.fragment_android, null);
        Button buttonAndroid = (Button) contentView.findViewById(R.id.buttonAndroid);
        buttonAndroid.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    Hotpatch.instance.installPatch("com.multiapk.modules.mobile.ios", 1, new File("/storage/emulated/0/ios.patch"));
                    Toast.makeText(mFragmentActivity, "合成成功", Toast.LENGTH_SHORT).show();
                } catch (Exception e) {
                    e.printStackTrace();
                    Toast.makeText(mFragmentActivity, "合成失败", Toast.LENGTH_SHORT).show();
                }
            }
        });
        return contentView;
    }
}

File: android/app/modules/mobile.android/src/main/java/com.multiapk.modules.mobile.android/AndroidFragment.kt
package com.multiapk.modules.mobile.android

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import com.mlibrary.multiapk.core.hotpatch.Hotpatch
import com.multiapk.library.base.CMFragment
import org.jetbrains.anko.toast
import java.io.File

class AndroidFragment : CMFragment() {
    @SuppressLint("InflateParams")
    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val contentView = inflater!!.inflate(R.layout.fragment_android, null)
        val buttonAndroid = contentView.findViewById(R.id.buttonAndroid) as Button
        buttonAndroid.setOnClickListener {
            try {
                Hotpatch.instance.installPatch("com.multiapk.modules.mobile.ios", 1, File("/storage/emulated/0/ios.patch"))
                context?.toast("合成成功")
            } catch (e: Exception) {
                e.printStackTrace()
                context?.toast("合成失败")
            }
        }
        return contentView
    }
}
